### ISSUE TITLE
Add support for .tgz suffix

### ISSUE THREAD

**Issue by Raboo** [engineerd/configurator#22]
Hi

Some projects uses a ".tgz" file name suffix for gzipped tar archives.
Could you possibly add support for .tgz suffix for archive extraction?

**Comment by Raboo** [comment]
For anyone that needs a work-around. Since the file suffix is read from the URL and not he actual file. This can be used to trick the code. 
```
https://github.com/vmware-tanzu/buildkit-cli-for-kubectl/releases/download/{{version}}/linux-{{version}}.tgz?.tar.gz
```
i.e. add `?.tar.gz` to the end of your url.

**Comment by radu-matei** [comment]
Implemented in #27
